extension/scripts/features/page-jump/ttPageJump.js

```diff
@@ -0,0 +1,698 @@
+			viewBox: "-2 0 17 17",
+		},
+		russianRoulette: {
+			d: "m15.36,5.76c-.36-1.24-1.02-2.37-1.91-3.3-1.62.11-3.15-.77-3.87-2.23C8.34-.08,7.03-.08,5.78.23c-.71,1.46-2.24,2.35-3.87,2.23-.89.93-1.55,2.06-1.91,3.3.91,1.35.91,3.12,0,4.47.36,1.24,1.02,2.37,1.91,3.3,1.62-.11,3.15.77,3.87,2.24,1.25.31,2.56.31,3.81,0,.71-1.46,2.24-2.35,3.87-2.23.89-.93,1.55-2.06,1.91-3.3-.91-1.35-.91-3.12,0-4.47h0Zm-5.08-1.27c.55-.96,1.78-1.28,2.73-.73.96.55,1.28,1.78.73,2.73-.55.96-1.78,1.28-2.73.73,0,0,0,0,0,0-.96-.55-1.28-1.78-.73-2.73Zm-5.2,7c-.55.96-1.78,1.28-2.73.73s-1.28-1.78-.73-2.73c.55-.96,1.78-1.28,2.73-.73,0,0,0,0,0,0,.96.55,1.28,1.78.73,2.73m-1.73-4c-1.1,0-2-.9-2-2s.9-2,2-2,2,.9,2,2-.9,2-2,2m5.33,7.23c-.96.55-2.18.22-2.73-.73-.55-.96-.22-2.18.73-2.73s2.18-.22,2.73.73c0,0,0,0,0,0,.55.96.22,2.18-.73,2.73m-2-6.73c0-.55.45-1,1-1s1,.45,1,1-.45,1-1,1-1-.45-1-1m2-3.27c-.96.55-2.18.22-2.73-.73-.55-.96-.22-2.18.73-2.73.96-.55,2.18-.22,2.73.73h0c.55.96.22,2.18-.73,2.73m3.33,7.77c-1.1,0-2-.9-2-2s.9-2,2-2,2,.9,2,2-.9,2-2,2",
+		},
+		spinTheWheel: {
+			d: "m12.35,12.68l.96,2.99c.04.14-.03.28-.17.33-.09.03-.19,0-.26-.06l-2.17-2.09c-2.12,1.11-4.65,1.11-6.77,0l-2.17,2.09c-.1.1-.27.09-.37-.01-.06-.07-.09-.17-.06-.26l.96-2.99C-.64,9.91-.79,5.27,1.99,2.32c2.77-2.95,7.41-3.1,10.37-.33,2.95,2.77,3.1,7.41.33,10.37-.11.11-.22.22-.33.33Zm-5.02.66c3.31,0,6-2.69,6-6S10.65,1.34,7.33,1.34,1.33,4.03,1.33,7.34c0,3.31,2.69,6,6,6Zm4.67-6c0,2.58-2.09,4.67-4.67,4.67s-4.67-2.09-4.67-4.67c0-2.58,2.09-4.67,4.67-4.67h0c2.58,0,4.67,2.09,4.67,4.67h0Z",
+		},
+	};
+
+	const pageRegistry = [
+		{ name: "Home", page: "/index.php", svg: svgIcons.home },
+		{ name: "Messages", page: "/messages.php", svg: svgIcons.messages },
+		{ name: "Events", page: "/page.php?sid=events", svg: svgIcons.events },
+		{ name: "Awards", page: "/awards.php", svg: svgIcons.awards },
+		{ name: "Items", page: "/item.php", svg: svgIcons.items },
+		{ name: "Bazaar", page: "/bazaar.php", svg: svgIcons.bazaar },
+		{ name: "Manage Bazaar", page: "/bazaar.php#/manage", svg: svgIcons.manageBazaar },
+		{ name: "Job", page: "/jobs.php", svg: svgIcons.job },
+		{ name: "Trades", page: "/trade.php", svg: svgIcons.trades },
+		{ name: "Crimes", page: "/crimes.php", svg: svgIcons.crimes },
+		{ name: "City", page: "/city.php", svg: svgIcons.city, filterBySubtext: "City" },
+		{ name: "Newspaper", page: "/newspaper.php", svg: svgIcons.newspaper },
+		{ name: "Forums", page: "/forums.php", svg: svgIcons.forums },
+		{ name: "Hall of Fame", page: "/halloffame.php", svg: svgIcons.hallOfFame },
+		{ name: "My Faction", page: "/factions.php?step=your", svg: svgIcons.hallOfFame },
+		{ name: "Recruit Citizens", page: "/bringafriend.php", svg: svgIcons.recruitCitizens },
+		{ name: "Calendar", page: "/calendar.php", svg: svgIcons.calendar },
+		{ name: "Mr & Ms Torn", page: "/competition.php", svg: svgIcons.mrMsTorn },
```

Comment:
Whilst the current competition is Mr & Ms Torn, this url is also reused in other competitions (from memory) such as dog tags and elim. May be easier to just call it something like "Competition" to prevent having to change this every couple months a new competition rolls around? Or even add multiple aliases for the same page for other competitions as well?

Reply:
Ahhh, thank you! I definitely overlooked that. Let me see if I can find a list of competitions, and then I can create multiple entries that all point to `competition.php`.

Reply:
Updated! I grabbed all the events from [here](https://wiki.torn.com/wiki/Annual_Competitions). Furthermore, depending on the current month, only the appropriate row will be visible in the results.